finality-aleph/src/network/session/manager.rs

```diff
@@ -7,13 +7,14 @@ use std::{
 use futures::channel::mpsc;
 use log::{debug, info};
 
+use super::Authentication;
```

Comment:
Ah, we generally avoid `super` (not sure whether that's actually good, but this is the current convention). In particular some stuff below is imported from `super`, but without using the keyword, so this is now inconsistent even within this one file.

Reply:
fixed